PXWeb/Resources/Languages/pxlang.sv.xml

```diff
@@ -1206,4 +1206,10 @@
   <sentence name="PxWebAdminUsersErrorMail" value="E-posten är inte korrekt ifylld" />
   <sentence name="PxWebAdminUsersLogin" value="Logga in" />
   <sentence name="PxWebAdminUsersAddedUser" value="har lagts till som användare" />
+  <sentence name="PxWebBulkInformation" value="&lt;p&gt;Tabellerna som finns tillgänglig för nedladdning innehåller hela datamängden och alla referensperioder som finns publicerad som statistik inom High Value Dataset.&lt;br/&gt;Filerna är tillgängliga i CSV-format och på grund av storleken på filerna är de packeterade i formatet ZIP.&lt;br/&gt;&lt;br/&gt;Frågor om innehållet besvaras av &lt;a href='https://www.scb.se/om-scb/kontakta-oss/statistikservice/'>Statistikservice&lt;/a&gt;&lt;/p&gt;" />
```

Comment:
This text should be more generic in PxWeb and not mention the HVD:s since the bulk download functionallity could be used for any type of data not only HVD:s

Reply:
Update text content in pxlang files. Removed text about HVD